skopt/tests/test_optimizer.py

```diff
@@ -194,3 +196,21 @@ def test_optimizer_base_estimator_string_smoke(base_estimator):
     opt = Optimizer([(-2.0, 2.0)], base_estimator=base_estimator,
                     n_initial_points=1, acq_func="EI")
     opt.run(func=lambda x: x[0]**2, n_iter=3)
+
+
+def test_defaults_are_equivalent():
```

Comment:
I'd delay this to an issue: the code for checking that the points sampled are the same gets a bit ugly if you have strings (I know lazy Tim) and we should think about how to run this test with less than 15 iterations when we add different seeds as starting point.